src/dom/Null.ts

```diff
@@ -0,0 +1,50 @@
+import {DomValue} from "./DomValue";
+import {IonType, IonTypes} from "../Ion";
+
+/**
+ * Represents a null[1] value in an Ion stream.
+ *
+ * An Ion null differs from a Javascript null in that an instance of an Ion null may have data associated with it.
+ *
+ * In particular, Ion nulls have an Ion data type:
+ *
+ *      text encoding | Ion data type
+ *  ======================================
+ *      null          | IonType.NULL (implicit)
+ *      null.null     | IonType.NULL (explicit)
```

Comment:
What is *implicit* vs. *explicit* mean here?  `null.null` and `null` are merely syntactic ways to spell the same thing, this doc seems to imply something more, but  I can't think of what could be meant.